front-end/README.md

```diff
@@ -0,0 +1,47 @@
+# Front end Example for Okta Spring Security Resource Server
+
+This front-end sample application can be used to test the [Resource Server](../resource-server) sample application provided in this repo.
+
+## Prerequisites
+
+Before running this sample, you should start the resource server backend.
+
+**backend:**
+```bash
+cd resource-server
```

Comment:
This *might* be a bit confusing, as `resource-server` isn't a subdirectory. Maybe update this to `cd ../resource-server`